clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -4431,6 +4431,26 @@ def AssumeSepStorageOp : CIR_Op<"assume.separate_storage", [SameTypeOperands]> {
   }];
 }
 
+//===----------------------------------------------------------------------===//
+// PtrMask Operations
+//===----------------------------------------------------------------------===//
+
+def PtrMaskOp : CIR_Op<"ptr_mask"> {
+  let summary = "Corresponding to LLVM's ptr mask operation";
```

Comment:
How about `summary = "Masks out bits of the pointer according to a mask"`? In the description you can add a note that this is similar to LLVM's operation.